openapi/paths/quotes@{id}@cancel.yaml

```
@@ -0,0 +1,26 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+post:
+  x-products:
+    - Core
+  tags:
+    - Quotes
+  summary: Cancel a quote
+  operationId: PostQuoteCancellation
+  x-sdk-operation-name: cancel
+  description: Cancels a draft, or issued, quote with a specified ID.
```

suggestion: suggestion for readability (I believe I may have added these comments in an earlier comment, sorry!)
```suggestion
  description: Cancels a draft or issued quote with a specified ID.
```